soyiyeon/백준_2523번문제.py

```diff
@@ -0,0 +1,13 @@
+# 예제를 보고 규칙을 유추한 뒤에 별을 찍어 보세요.
+
+# 첫째 줄에 N(1 ≤ N ≤ 100)이 주어진다.
+
+N = int(input())
+
+for j in range(1, N):
+    print("*"*(j))
```

Comment:
```suggestion
    print("*" * j)
```
괄호를 사용하지 않아도 좋을 것 같습니다!